plugins/aws/access_key.go

```diff
@@ -59,6 +59,11 @@ func AccessKey() schema.CredentialType {
 				MarkdownDescription: "ARN of the MFA serial number to use to generate temporary STS credentials if the item contains a TOTP setup.",
 				Optional:            true,
 			},
+			{
+				Name:                fieldname.Role,
+				MarkdownDescription: "ARN of the Role to be assumed.",
+				Optional:            true,
+			},
```

Comment:
Hello 👋! I'm avidly waiting for this feature, and I wanted to give my 2 cents about this 🙂. 

In all my experiences with AWS, we always had multiple roles (across multiple AWS accounts) with the same credentials. As a matter of fact, it's considered a best practice.
Furthermore, while it's possible to duplicate the credentials in 1P for each role, it will be cumbersome to rotate them all. 

In other words, I think having only one possible role per AWS credentials would considerably lessen the usefulness of handling AWS roles in 1Password 🙂.

Reply:
It would make sense yes to have multiple roles. I am fine with both :) Just want to get rid of my awsume locally 😅

Reply:
Is there any progress? :)

Reply:
I agree that for this functionality to be complete, we need to support multiple roles.

We'll also need to figure out the experience of selecting the role to use. As @ahamez already noted, it's common practice to use the same credentials for multiple roles. Therefore, we should make it possible to select/configure the role separately from the credential. For example, you might want to configure a global default for the credentials, while specifying the role in specific scripts.

I'm wondering what provides the best experience for role selection. A couple examples:
1. 1Password prompt: roles clearly defined in 1Password, but also an extra step in your workflow
2. `AWS_ROLE_ARN` envvar: role defined in the script that uses it, no prompt/distraction when executing the script, role not visible in 1Password

Reply:
@ahamez how are you currently selecting the role to use with AWS? How do you like that? Are there any problems you run into with that? How would you prefer this to work? How would you prefer this to work with 1Password?

Reply:
@SimonBarendse Currently, we use aws-vault to assume roles and have credentials secured. For instance:
```
aws-vault exec <ROLE> -- aws ecr get-login-password --region REGION | docker login --username AWS --password-stdin REGISTRY
```

However, when working with multiple computers, it's annoying as I have to setup it each time. But, most importantly, as we use 1Password for all other workflows, we would have a unified way of securing all workflows, with the added benefit of easily rotating credentials in only place.

As for a way of using this, I think we would be perfectly fine with something like:
```
op plugin aws <ROLE> -- ...
```
As long as we're able to select the role on the command line, it would be 👌.

Reply:
Thank you